Compiler/types.py

```diff
@@ -2436,13 +2449,17 @@ def __init__(self, rows, columns):
         self.rows = rows
         self.columns = columns
         self.multi_array = MultiArray([rows, columns, 4], sint)
+        
+    def delete(self):
+        self.multi_array.delete()
 
     def __getitem__(self, index):
         return sfloatArray(self.columns, self.multi_array[index].address)
 
 class sfixArray(Array):
     def __init__(self, length, address=None):
         self.array = Array(length, sint, address)
+        self.address = self.array.address
```

Comment:
This seems redundant.

Reply:
Either we make `address` available to `self` such that the inherited `Array.delete()` function works or we call delete on `self.array`. Now that you mention it, last looks better.